src/main/java/cinema/model/mapper/MovieSessionMapper.java

```diff
@@ -0,0 +1,38 @@
+package cinema.model.mapper;
+
+import cinema.model.CinemaHall;
+import cinema.model.Movie;
+import cinema.model.MovieSession;
+import cinema.model.dto.MovieSessionRequestDto;
+import cinema.model.dto.MovieSessionResponseDto;
+import cinema.service.CinemaHallService;
+import cinema.service.MovieService;
+import java.time.LocalDateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapper {
+    @Autowired
+    private MovieService movieService;
+
+    @Autowired
+    private CinemaHallService cinemaHallService;
+
+    public MovieSession getMovieSessionFromMovieSessionRequestDto(
+            MovieSessionRequestDto movieSessionRequestDto) {
+        Movie movie = movieService.getById(movieSessionRequestDto.getMovieId());
+        CinemaHall cinemaHall = cinemaHallService.getById(
+                movieSessionRequestDto.getCinemaHallId());
+        LocalDateTime showtime = LocalDateTime.parse(movieSessionRequestDto.getTime());
+        return new MovieSession(movie,cinemaHall,showtime);
```

Comment:
```suggestion
        return new MovieSession(movie, cinemaHall, showtime);
```
1) don't forget to run mvn clean package before pushing something to git;
2)let's use setters))

Reply:
I always do that on a regular basis, but you always find one single syntax mistake :)